sdk/mediaservices/Microsoft.Azure.Management.Media/CHANGELOG.md

```diff
@@ -0,0 +1,42 @@
+﻿# Microsoft.Azure.Management.Media release notes
```

Comment:
Please update AzureSDK.RP.props with the new API-Version.

Reply:
That file says its autogenerated.. not sure why its not generating 
![image](https://user-images.githubusercontent.com/5192456/108584968-374b3c00-72fa-11eb-8c40-b991ba9b9138.png)